### ISSUE TITLE
awesome

### ISSUE THREAD

**Issue by EdgeEden** [TMFfa/practice#1]
All that stuff created by yourself? That's awesome
Waiting 4 releases

**Comment by EdgeEden** [comment]
And more description!

**Comment by faf4r** [comment]
well, most of that was created by me to practise . While some source which was copied from Internet.




------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2021年4月10日(星期六) 晚上10:07
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; 
主题: [TMFfa/practice] awesome (#1)





 
All that stuff created by yourself? That's awesome
 Waiting 4 releases

**Comment by faf4r** [comment]
The reason I submit these source is to store my practices.
As they are some gadgets, I didn't make any description and no more description!(You know that I had spent a long time upload these source.)
If you want to get some details about the source, just read it by yourself referring to the file name.




------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2021年4月10日(星期六) 晚上10:07
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; 
主题: Re: [TMFfa/practice] awesome (#1)





 
And more description!

**Comment by EdgeEden** [comment]
Yeah maybe I would try it out and pull request for description myself233

**Comment by faf4r** [comment]
That couldn't be better!
Besides, among those projects, [these project](https://github.com/TMFfa/practice/tree/main/tuacg) was written today to download some images for little egg.
And I have to say that some files may be unused or confused, those are some notes or data created when writing, which you can just ignore them.

**Comment by EdgeEden** [comment]
Little egg hhh
ok I will try to fix that if I  can
